src/python.gen.ts

```diff
@@ -367,6 +366,30 @@ ${this.hooks.join('\n')}
       + `${returnStmt}`
   }
 
+  bodyParamsTypeAssertions(indent: string, bodyParams: IParameter[]): string {
+    const bump = indent + this.indentStr
+    let assertions: string = ''
+    if (bodyParams.length > 0) {
+      for (const param of bodyParams) {
+        if (param.location === strBody) {
+          let conditionStr = param.required ? '' : `${indent}if ${param.name}:\n${bump}`
+          let type = this.writeableType(param.type) || param.type
+          let bodyType = this.typeMapMethods(type).name
+          if (bodyType.startsWith('Sequence')) {
+            bodyType = 'Sequence'
+          } else if (bodyType.startsWith('MutableMapping')) {
+            bodyType = 'MutableMapping'
+          } else if (bodyType.startsWith('models.DelimSequence')) {
+            bodyType = 'models.DelimSequence'
+          }
+          assertions += (`${conditionStr}${indent}assert isinstance(${param.name}, ${bodyType})`)
```

Comment:
Given my comment above I don't think there's ever more than 1 element in `bodyParams`, however, if there were I think you want to add a `\n` here and remove L388